libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/voice/NavigationInstructionPlayer.java

```diff
@@ -35,44 +37,36 @@ private void initAudioFocusRequest() {
     }
   }
 
-  private void initInstructionPlayer(Context context, @Nullable String awsPoolId,
-                                     Locale locale) {
-    if (!TextUtils.isEmpty(awsPoolId)) {
-      instructionPlayer = new PollyPlayer(context, awsPoolId, locale);
-      isPollyPlayer = true;
-    } else {
-      instructionPlayer = new DefaultPlayer(context, locale);
-    }
-    instructionPlayer.addInstructionListener(this);
+  private void initInstructionPlayers(Context context, Locale locale) {
+    mapboxSpeechPlayer = new MapboxSpeechPlayer(context, locale, this);
+    androidSpeechPlayer = new AndroidSpeechPlayer(context, locale, this);
   }
 
-  @Override
-  public void play(String instruction) {
-    instructionPlayer.play(instruction);
+  public void play(VoiceInstructionMilestone voiceInstructionMilestone) {
+    instructionQueue.add(voiceInstructionMilestone);
+    mapboxSpeechPlayer.play(voiceInstructionMilestone.getSsmlAnnouncement());
   }
 
-  @Override
   public boolean isMuted() {
-    return instructionPlayer.isMuted();
+    return isMuted;
   }
 
-  @Override
   public void setMuted(boolean isMuted) {
-    instructionPlayer.setMuted(isMuted);
+    this.isMuted = isMuted;
+    mapboxSpeechPlayer.setMuted(isMuted);
+    androidSpeechPlayer.setMuted(isMuted);
   }
 
-  @Override
   public void onOffRoute() {
```

Comment:
Should we clear the instruction queue here too?  That way it stays aligned with the two players.  Or is this being done elsewhere and I'm not 👀

Reply:
This has been done, not sure why this comment is still showing